test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/TestCommand.cs

```diff
@@ -15,6 +17,8 @@ public class TestCommand
 
         public string WorkingDirectory { get; set; }
 
+        public Process CurrentProcess { get; set; }
```

Comment:
I don't feel that this is right that the TestCommand class holds the state of a running process. What if we wanted to execute two of the same to test concurrency issues?